grakn-graph/src/test/java/ai/grakn/generator/GraknGraphs.java

```diff
@@ -147,30 +147,40 @@ public GraknGraphs setOpen(boolean open) {
     private final ImmutableList<Runnable> mutators = ImmutableList.of(
             () -> {
                 TypeName typeName = typeName();
-                EntityType entityType = graph.putEntityType(typeName);
+                EntityType superType = entityType();
+                EntityType entityType = graph.putEntityType(typeName).superType(superType);
```

Comment:
You're totally right - all of these methods can fail. I have a `try...catch` wrapping these at line 92 that catches expected exception types. If there's a failure, it will try another random mutation.

I considered checking the arguments ahead of time, but it felt like replicating the logic and didn't seem worth it - given the result is the same.